src/main/java/ai/elimu/web/content/word/WordEditController.java

```diff
@@ -165,7 +166,7 @@ public String handleRequest(
         // TODO: labeled Videos
         
         // Look up StoryBook Paragraphs that contain this Word
-        List<StoryBookParagraph> storyBookParagraphsContainingWord = storyBookParagraphDao.readAllContainingWord(word.getText());
+        List<StoryBookParagraph> storyBookParagraphsContainingWord = storyBookParagraphDao.readAllContainingWord(StringEscapeUtils.escapeSql(word.getText()));
```

Comment:
@tomaszsmy I suggest moving this one layer down, to the DAO implementation itself: `StoryBookParagraphDaoJpa`